cli/src/commands/abstract_build_docker.py

```diff
@@ -3,6 +3,21 @@
 
 class AbstractBuildDocker(AbstractCommand):
     @staticmethod
-    def build_docker_file(file, image="algolia/docsearch-scraper-dev"):
-        cmd = ["docker", "build", "-t", image, "-f", file, "."]
+    def build_docker_file(file, image="algolia/docsearch-scraper-dev",
+                          local_tag=False):
+        tags = [image]
+
+        if local_tag:
+            from subprocess import check_output
+            tag = check_output(
```

Comment:
Please put this in a separate function: it can quickly grow quite big (thinking of error handling/recovery) and we don't want to see this being done here :)